tests/08-native-runs/18-bitrev-test.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+source ../utils.sh
+
+# Contiki directory
+CONTIKI=$1
+
+# Example code directory
+CODE_DIR=$CONTIKI/tests/08-native-runs/18-bitrev/
+CODE=test-bitrev
+
+# Starting Contiki-NG native node
+echo "Starting native node for bit reversal test"
+make -C $CODE_DIR TARGET=native > make.log 2> make.err
+$CODE_DIR/$CODE.native > $CODE.log 2> $CODE.err &
```

Comment:
Contiki-NG no longer copies generated artifacts outside build directory by default.
```suggestion
$CODE_DIR/build/native/$CODE.native > $CODE.log 2> $CODE.err &
```